js/views/account.js

```diff
@@ -65,6 +65,10 @@ define(function(require) {
 		toggleCollapse: function() {
 			this.collapsed = !this.collapsed;
 			this.render();
+		},
+		toggleMenu: function(e) {
+			e.preventDefault();
+			this.menuShown = !this.menuShown;
 			this.listenTo(Radio.ui, 'document:click', function(event) {
```

Comment:
add a ``onShow`` method and move that event handler registration in there. That method is called by Marionette once a view has finished rendering and is added to the DOM.